01-python-basics/hw-01-python-basics.py

```diff
@@ -0,0 +1,51 @@
+# import lib with needed functions
+import random
+
+# create list of 100 random numbers from 0 to 1000
+# using sample method that returns a list of 100 elements in range between 0 and 1000
+random_list = random.sample(range(0, 1000), 100)
+
+# sort list from min to max (without using sort())
+# go through each element of list
+for i in range(len(random_list)):
```

Comment:
In this case will have 100 cycles even for already sorted list
To avoid it suggest add lines:
while sorted_count > 0:
sorted_count = 0

and then:
sorted_count = sorted_count + 1

Reply:
need some more details :)